.github/workflows/python-bdist.yml

```diff
@@ -0,0 +1,135 @@
+name: Python Binary Distribution
+
+on: [push]
```

Comment:
Think we need to add [release](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release) as well